src/Moryx.ControlSystem/Activities/ActivityClassification.cs

```diff
@@ -33,6 +33,7 @@ public enum ActivityClassification
         /// <summary>
         /// Activity performs a preparation, for example for a <see cref="Production"/> activity
         /// </summary>
-        Preparation = 0x08
+        Preparation = 0x08,
+        Unknown = 0x10
```

Comment:
A comment would be helpful. Unknow could also be the default 0 as it is currently not used.